src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml

```diff
@@ -53,6 +53,12 @@ limitations under the License.
         <!-- RESERVED = 0x03 -->
     </struct>
 
+    <enum name="OperatingModeEnum" type="enum8">
```

Comment:
pending for spec change on https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8736